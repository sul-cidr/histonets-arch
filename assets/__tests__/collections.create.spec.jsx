import React from "react";
import { shallow } from "enzyme";
import App from "../collections.create";

describe("<App>", () => {
  it("renders as expected", () => {
    const component = shallow(<App uploadUrl="/" imageFormats={['png', 'jpg']} />);
    expect(component).toMatchSnapshot();
  });
});
