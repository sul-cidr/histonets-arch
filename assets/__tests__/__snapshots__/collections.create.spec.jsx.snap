// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> renders as expected 1`] = `
<div>
  <CollectionCreate
    imageFormats={
      Array [
        "png",
        "jpg",
      ]
    }
    images={Array []}
    imagesId="id"
    imagesName="images"
    uploadUrl="/"
  />
</div>
`;
